frontend/pulse3d/src/utils/generic.js

```diff
@@ -101,6 +102,14 @@ const getWaveformCoordsFromTable = async (table, normalizeYAxis) => {
   return coordinatesObj;
 };
 
+const getTableFromParquet = async (buffer) => {
+  const wasmModule = await import("parquet-wasm/esm/arrow1.js");
```

Comment:
Is it possible to import this module just once? Could you move this out into the global scope and import?

Reply:
No that causes issues, needs to be awaited in react